core/server/master/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -41,9 +43,13 @@
   int getLostWorkerCount();
 
   /**
+   * Gets the worker information list.
+   *
+   * @param options the WorkerInfoOptions defines the info range
    * @return a list of {@link WorkerInfo} objects representing the workers in Alluxio
    */
-  List<WorkerInfo> getWorkerInfoList() throws UnavailableException;
+  List<WorkerInfo> getWorkerInfoList(WorkerInfoOptions options)
```

Comment:
NIT: should `WorkerInfoOptions` match the method, like `GetWorkInfoListOptions`?

Reply:
GetWorkInfoListOptions is clearer for future contributors.
Changed to GetWorkerInfoListOptions.
Thanks for the suggestion.